docs/develop/dapps/ton-connect/message-builders.mdx

```diff
@@ -440,11 +440,11 @@ The `messageBody` for Jetton Transfer([TEP-74](https://github.com/ton-blockchain
     const body = beginCell()
         .storeUint(0xf8a7ea5, 32) // opcode for jetton transfer
         .storeUint(0, 64) // query id
-        .storeCoins(toNano(5)) // jetton amount, amount * 10^9
+        .storeCoins(toNano("5")) // jetton amount, amount * 10^9
```

Comment:
Let's add clarification, that decimal size individually specified for every jetton by its master contract

Reply:
ok, I'll fix everything soon